From a30e4077e7074ca745236ebe19e30cd3978de8ed Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Tue, 28 Feb 2017 19:22:27 +0200
Subject: [PATCH 0823/1345] fix: dts: cp110: sata: set comreset_u and comwake
 values

commit  e9e09014fdb8a729e1a0458ddc6115802172475b from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

The COMRESET and COMWAKE upper limit updated values
improve SATA electical behavior of cp110 SATA controller.

Change-Id: I6a773c4056212baee6960c2e78cfa7fe803c22e9
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37098
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi |    2 ++
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi   |    2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
index e68c416..79529d2 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
@@ -70,6 +70,8 @@ sata@540000 {
 	interrupts = <ICU_GRP_NSR 107 IRQ_TYPE_LEVEL_HIGH>;
 	clocks = <&cps_syscon0 1 15>;
 	status = "disabled";
+	comreset_u = <0x17>;
+	comwake = <0x6>;
 	#address-cells = <1>;
 	#size-cells = <0>;
 
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index eb53aa7..868904c 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -152,6 +152,8 @@ sata: sata@540000 {
 	dma-coherent;
 	interrupts = <ICU_GRP_NSR 107 IRQ_TYPE_LEVEL_HIGH>;
 	clocks = <&cpm_syscon0 1 15>;
+	comreset_u = <0x17>;
+	comwake = <0x6>;
 	status = "disabled";
 	#address-cells = <1>;
 	#size-cells = <0>;
-- 
1.7.9.5

